schema.sql

```diff
@@ -1,5 +1,17 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+/* Create database */
+CREATE DATABASE vet_clinic ;
+
+-- connect database 
+\c vet_clinic
+
+/* create table */
+ CREATE TABLE animals(
+vet_clinic(# id                 INT GENERATED BY DEFAULT AS IDENTITY,
+vet_clinic(# name               VARCHAR(100) NOT NULL,
+vet_clinic(# date_of_birth      DATE NOT NULL,
+vet_clinic(# escape_attempts    INT,
+vet_clinic(# neutered           BIT,
+vet_clinic(# weight_kg          DECIMAL) ;
+
```

Comment:
- [ ] Initially i was unable to test your queries for this milestone as i cannot create the animals table using the highlighted code, however i made the necessary corrections to this file using the below code snippet below and i was able to test your project to ensure that every schema, insertion or query works appropriately i would need you to update this piece of code so the anyone else trying to test would see and can confirm that everything works properly, this is the only thing you need to change as everything else works properly, Good job :+1: :sparkles: 
- code snippet
```sql
CREATE TABLE animals(
  id                 INT GENERATED BY DEFAULT AS IDENTITY,
  name               VARCHAR(100) NOT NULL,
  date_of_birth      DATE NOT NULL,
  escape_attempts    INT,
  neutered           BIT,
  weight_kg          DECIMAL) ;
```

Reply:
@Efiamotu-1  I am yet to understand the updated version of the code. Can you please help out in detail?

Reply:
How do I update the table? @Efiamotu-1

Reply:
@Efiamotu-1, do I need to update the code in the schema.sql file?

Reply:
@efmomoh the code you had cannot be used to creat a table, basically which is why i gave you the corrected code which works

Reply:
Thank you @Efiamotu-1  for the clarity 🙏